website/static/js/fangorn.js

```diff
@@ -455,13 +476,14 @@ function doItemOp(operation, to, from, rename, conflict) {
             tb.modal.update(mithrilContent, mithrilButtons);
             return;
         }
-        from.data = resp;
+        from.data = resp.data.attributes;
         from.data.status = undefined;
         from.notify.update('Successfully ' + operation.passed + '.', 'success', null, 1000);
 
+    var fromName = from.data.name || from.data.attributes.name;
```

Comment:
I'm gonna work on setting up my vim for .js as soon as these PRs are in the clear. I cut out everything .js related after the tabs issue. So, just running base vim for .js at the moment. Sorry for all the styling issues.